app/forms/session_wizard_steps/base.rb

```diff
@@ -0,0 +1,50 @@
+module SessionWizardSteps
+  class Base
+    include ActiveModel::Model
+    include ActiveModel::Validations::Callbacks
+
+    attr_accessor :wizard
+
+    def self.permitted_params
+      []
+    end
+
```

Comment:
can we get rid of the methods that aren't used by `SessionWizardSteps::SignIn` and `SessionWizardSteps::SignInCode` ?

Reply:
I've made it raise NotImplementedError instead